transactions.asciidoc

```diff
@@ -489,7 +489,7 @@ https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
 
 === The signature prefix value (v) and public key recovery
 
-As mentioned in <<tx_struct>>, the transaction message doesn't include any "from" field. That's because the originator's public key can be computed directly from the ECDSA signature. Once you have the public key, you can compute the address easily. The process of recovering the signer's public key is called a _Publik Key Recovery_.
+As mentioned in <<tx_struct>>, the transaction message doesn't include any "from" field. That's because the originator's public key can be computed directly from the ECDSA signature. Once you have the public key, you can compute the address easily. The process of recovering the signer's public key is called a _Public Key Recovery_.
```

Comment:
I recommend removing the word "easily". I doubt it will seem easy to the average reader (unless you show them how easy it is).